molo/profiles/templates/profiles/change_password.html

```diff
@@ -2,41 +2,43 @@
 {% load i18n %}
 
 {% block content %}
-<div class="form-wrap change-password">
-<form method="post" id="frmProfilePasswordChange" action="" class="profile">
+<div class="profiles">
+<form method="post" action="" class="profiles__form">
 {% csrf_token %}
    {% if form.errors or form.non_field_errors %}
       {% if form.non_field_errors %}
           {% for error in form.non_field_errors %}
-              <p class="errorlist">{{ error }}</p>
+              <span class="error">{{ error }}</span>
           {% endfor %}
       {% endif %}
   {% endif %}
 
   {% if messages %}
-    <ul class="errorlist">
+    <ul class="error-list">
         {% for message in messages %}
-        <li>{{ message }}</li>
+        <li class="error-list__item error">{{ message }}</li>
         {% endfor %}
     </ul>
   {% endif %}
 
-   <div>
-      <label class="label-wide">{% trans "Old Password:" %}</label>
-      <p>{{form.old_password}}</p>
-      <span>{{form.old_password.errors}}</span>
-   </div>
-   <div>
-      <label class="label-wide">{% trans "New Password:" %}</label>
-      <p>{{form.new_password}}</p>
-      <span>{{form.new_password.errors}}</span>
-   </div>
-   <div>
-      <label class="label-wide">{% trans "Confirm New Password:" %}</label>
-      <p>{{form.confirm_password}}</p>
-      <span>{{form.confirm_password.errors}}</span>
-   </div>
-   <input type="submit" value='{% trans "Change Password" %}' class="btn">
+  <fieldset>
+      <label>{% trans "Old Password" %}</label>
+      {{form.old_password}}
+      {{form.old_password.errors}}
+  </fieldset>
+
+  <fieldset>
+      <label>{% trans "New Password" %}</label>
```

Comment:
The `label` tags should have a `for` attribute -- I know that Django can also generate the output of the `label` tags. I just don't know if we then have the ability to control the label's value.